api/docs/v2/storefront/index.yaml

```diff
@@ -8,6 +8,8 @@ info:
 paths:
   /cart/add_item:
```

Comment:
Could you add other endpoints that were merged? We already have create, empty, remove_line_item and I'm working on set_quantity endpoint

Reply:
@WitekL this PR is only for Products API